aiida_orca/__init__.py

```diff
@@ -1,3 +1,3 @@
 """AiiDA-Orca plugin"""
 
-__version__ = '0.5.1'
+__version__ = '1.0.0b'
```

Comment:
Are we jumping straight to 1.0 here? I would assume that v1 should incorporate all breaking changes and also support aiida 2.0. Perhaps we could release one more pre-1.0 version with support for ORCA5 and defer breaking changes for later?

Reply:
I started to version AiiDA 1 compatible ones with 1 and then AiiDA 2 would be aiida-orca 2 😉

Reply:
I down played it to `0.6.0` for now. I will increment to `1.0.0` once I fully replace `cclib` parsers.